doc/content.tex

```diff
@@ -1,1349 +0,0 @@
-            \end{center}
-        \end{figure}
-
-        After adding concrete chart predictions can be enabled by increasing prediction horizon input.
-        So there is no need to enable predictions manually by creating relationship in Inventory.
-
-        Predictive charts were used from repository \texttt{hawkular-charts}. These charts are built on top of
-        D3\footnote{Avalilable at \url{https://d3js.org/}.} library and encapsulated as Angular directives.
-
-    %%%%%%%%
-    \section{Test Automation and Documentation}
-    Core functionality of the application is covered by unit tests. Junit framework is used for this type of tests.
-    Because Data Mining interacts with other applications and modules integration tests are also implemented. These
-    tests are run from Maven profile which starts web server with necessary modules deployed.
-
-    The last type of implemented tests are end-to-end tests which calls REST endpoint of one module and expect result
-    in call from another module. This type of tests are also executed in Maven profile with running web server.
-    End-to-end tests for Data Mining project live in the main Hawkular integration repository\footnote{
-    Available at \url{https://github.com/hawkular/hawkular}.}.
-
-    All tests are automatically executed every time developer creates a pull request or pushes code to the master
-    branch. For this Travis continuous integration platform was used which is for free for open source projects on
-    Github.
-
-    Documentation is directly written in Java code as javadoc. REST API documentation is also directly
-    written in source code using Swagger framework and automatically generated at build time
-    \footnote{Available at \url{http://www.hawkular.org/docs/rest/rest-datamining.html}.}.
-
-    %%%%%%%%
-    \section{Releases}
-    By the time writing of this thesis there was only one release with version \texttt{0.1.Final}. It is also available
-    in the Maven central repository. Complete group id is \texttt{org.hawkular.datamining}. The most important
-    artifact ids are listed in table \ref{tab:datamining-modules}. Release notes are available on
-    Github\footnote{Available at \url{https://github.com/hawkular/hawkular-datamining/releases}.}.
-
-    %%%%%%%%
-    \section{Retrospective}
-    In this section it is mentioned how the project was changing during the implementation, what did not worked well and
-    what could have be done better.
-
-    At the beginning of the project Apache Spark\footnote{Open source cluster computing framework available at
-    \url{https://spark.apache.org/}.} was used for building regression models. Its machine learning library (MLIB)
-    offers several learning algorithms. In Data Mining linear regression with stochastic gradient descent was used.
-    Because the gradient is stochastic produced predictions are not very robust and quickly change by significant value.
```

Comment:
nejak mi tam ta ciarka nedava zmysel nema to byt skor takto?

``Because the gradient is stochastic, produced predictions are not very robust and quickly change by significant value.  ``